specification/elastic/resource-manager/Microsoft.Elastic/preview/2023-06-15-preview/elastic.json

```diff
@@ -0,0 +1,2581 @@
+        "operationId": "TrafficFilters_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MonitorNameParameter"
+          },
+          {
+            "in": "query",
+            "name": "rulesetId",
+            "description": "Ruleset Id of the filter",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderDefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "TrafficFilters_Delete": {
+            "$ref": "./examples/TrafficFilters_Delete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Elastic/getOrganizationApiKey": {
+      "post": {
+        "tags": [
+          "ApiKey"
+        ],
+        "summary": "Fetch User API Key from internal database, if it was generated and stored while creating the Elasticsearch Organization.",
+        "description": "Fetch User API Key from internal database, if it was generated and stored while creating the Elasticsearch Organization.",
+        "operationId": "Organizations_GetApiKey",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Email Id parameter of the User Organization, of which the API Key must be returned",
+            "schema": {
+              "$ref": "#/definitions/UserEmailId"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/UserApiKeyResponse"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderDefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Organizations_GetApiKey": {
+            "$ref": "./examples/Organizations_GetApiKey.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Elastic/elasticOrganizationToAzureSubscriptionMapping": {
+      "get": {
```

Comment:
>"get" [](http://example.com/codeflow?start=6&length=5)

if it is resource, it must have both Individual Get and Collection Get.
If it is not a resource, recommend to use "post"

Reply:
Updated